freqtrade/rpc/api_server/webserver.py

```diff
@@ -176,6 +244,8 @@ def start_api(self):
             if self._standalone:
                 self._server.run()
             else:
+                if self._config.get('api_server', {}).get('enable_message_ws', False):
+                    self.start_message_queue()
```

Comment:
what's the implication of starting the ws endpoints - but not the message queue?
wouldn't that mean that websockets will become non-functional? (maybe you just forgot to remove this part? if yes, don't forget the config option)

Reply:
Good catch

Reply:
It is already removed in the config option